docs/README.md

```diff
@@ -0,0 +1,19 @@
+# Welcome to Win32 App Isolation Introduction
+
+### Target Applications
+
+* Win32
+* Centennial apps.
+
+### Minimum requirements:
+
+* Windows insider version >= 25357
+* [MPT v1.2023.517.0] (https://github.com/microsoft/win32-app-isolation/releases/tag/v0.1.1)
+* If is needed to know which capabilities are required, one will use ACP (check the MPT link) and [WPR] (https://learn.microsoft.com/en-us/windows-hardware/test/wpt/windows-performance-recorder)
+
+### Creating a Silo App:
+
+[Detailed instructions] (https://github.com/microsoft/win32-app-isolation/blob/v0.1.1/docs/packaging/msix-packaging-tool.md)
+1.	Create an MSIX package from Win32 installer (if the app is not already MSIX)
```

Comment:
Replace the `\t` after `1.` (and `2.`, `3.`) with a single space.

We don't need the detailed instructions. We can put the link right after the description.

`1. Create an MSIX package from Win32 installer *if the app is not already MSIX* ([link](docs/packaging/msix-packaging-tool.md#win32---msix))`

Notice that:
1. I used `*text*` to make the text a different style (italic in this case).
2. I used *relative* path instead of *absolute* path because it would work on other forked repos as well.
3. A hastag `#win32--msix` is introduced to land on the corresponding section directly.